changelog/891.feature.rst

```diff
@@ -0,0 +1,2 @@
+Adds :attr:`HTTPException._errors` for parsing errors.
+Adds :meth:`_parse_sync_warning` and :meth:`_parse_options` more informative errors for command sync warnings.
```

Comment:
These are implementation details, it's fine if the changelog just mentions that the warning message was improved.

Reply:
+1
we can also provide an example here of the new warning message, and be clear about when this might be seen (and the benefit it provides)